specification/batch/data-plane/Microsoft.Batch/2017-09-01.6.0/BatchService.json

```diff
@@ -12206,6 +12249,25 @@
       },
       "title": "The result of listing the jobs in an account."
     },
+    "TaskContainerExecutionInformation": {
+      "properties": {
+        "containerId": {
+          "type": "string",
+          "title": "The ID of the container."
+        },
+        "state": {
+          "type": "string",
+          "title": "The state of the container.",
+          "description": "This is the state of the container according to the Docker service. It is equivilant to the status field returned by \"docker inspect\"."
+        },
+        "error": {
+          "type": "string",
+          "title": "Detailed error information about the container.",
+          "description": "This is the detailed error string from the Docker service, if available. It is equivilant to the error field returned by \"docker inspect\"."
+        }
+      },
+      "title": "Contains information about the container which the task is executing in."
```

Comment:
"in which the task is executing"

Reply:
picky: this is a type so we can't really say "the task" - perhaps e.g. "Information about a container in which a task is executing."?

Reply:
Fixed